lib/smart_city/dataset/technical.ex

```diff
@@ -77,6 +77,13 @@ defmodule SmartCity.Dataset.Technical do
     |> new()
   end
 
+  def new(%{dataName: _, orgName: _, systemName: _, sourceUrl: _, sourceFormat: type, schema: schema} = msg) do
+    msg
+    |> Map.put(:schema, Helpers.to_atom_keys(schema))
```

Comment:
Should we just atomize all keys again (just in case there are any other nested structures - now or in the future)?

Reply:
The way we do `to_atom_keys` does not play well with structs.

```
(Protocol.UndefinedError) protocol Enumerable not implemented for %SmartCity.Dataset.Technical{...
```